go.sum

```diff
@@ -1,13 +1,35 @@
 github.com/cenkalti/backoff v2.0.0+incompatible h1:5IIPUHhlnUZbcHQsQou5k1Tn58nJkeJL9U+ig5CHJbY=
```

Comment:
This file contains additions but no deletions even though some modules have been removed. Also, the build is failing most likely due to some dependency not being properly defined in this file that a docker instance is trying to append.

Reply:
re-ran `go get` to update the files